submarine-server/server-core/src/main/java/org/apache/submarine/server/SubmitterManager.java

```diff
@@ -92,11 +98,13 @@ private void loadSubmitters() {
     return urls.toArray(new URL[0]);
   }
 
-  /**
-   * Get the specified submitter by submitter type
-   * @return submitter
-   */
-  public JobSubmitter getSubmitterByType(String type) {
-    return submitterMap.get(type);
+  private static class JobSubmitterConfig {
+    private final String clazzName;
+    private final String classpath;
```

Comment:
Please change `classpath` to `classPath`.

Reply:
The [classpath](https://en.wikipedia.org/wiki/Classpath_(Java)) is a parameter in JVM, so I think we should keep this naming convention.